docs/guide/controller-programming.md

```diff
@@ -98,11 +243,49 @@ The call to the `wb_distance_sensor_get_value` function retrieves the latest val
 
 Note that some device return vector values instead of scalar values, for example these functions:
 
+%tab-component "language"
+%tab "C"
 ```c
 const double *wb_gps_get_values(WbDeviceTag tag);
 const double *wb_accelerometer_get_values(WbDeviceTag tag);
 const double *wb_gyro_get_values(WbDeviceTag tag);
 ```
+%tab-end
+
+%tab "C++"
+```cpp
+webots::GPS::const double *getValues() const;
+webots::Accelerometer::const double *getValues() const;
+webots::Gyro::const double *getValues() const;
```

Comment:
That looks odd... The following looks better, isn't it?
```suggestion
const double *webots::GPS::getValues() const;
const double *webots::Accelerometer::getValues() const;
const double *webots::Gyro::getValues() const;
```

Reply:
Ah yes my bad, this is what I wanted to do in the first place besides...